elements/pfe-progress-indicator/demo/pfe-progress-indicator.html

```diff
@@ -0,0 +1,43 @@
+<link rel="stylesheet" href="/core/pfe-core/pfe.min.css">
+<link rel="stylesheet" href="/core/pfe-styles/pfe-base.css">
+<link rel="stylesheet" href="/core/pfe-styles/pfe-context.css">
+<link rel="stylesheet" href="/core/pfe-styles/pfe-layouts.css">
```

Comment:
Not sure these stylesheets need to be included in the demo could not find the context in which they are used or needed.

```Suggestion
```